lib/parser.js

```diff
@@ -22,10 +17,10 @@ const constants = {
 
 const readParcel = (
   // Read parcel from buffer
-  buffer // [buffer type]
-  // Return parcel object
+  buffer // buffer
+  // Return: object
```

Comment:
This should be `Returns`. Check out my last review [comment](https://github.com/metarhia/metacom/pull/24#discussion_r216137358)